src/Api/Deployments.php

```diff
@@ -36,6 +36,11 @@ public function all($project_id, array $parameters = [])
         $resolver->setDefined('sort')
             ->setAllowedTypes('sort', 'string')
             ->setAllowedValues('sort', ['desc', 'asc']);
+        $resolver->setDefined('status')
```

Comment:
Please also update the phpdoc

Reply:
Ohh yea forgot about that, have updated the phpdoc with new values

Reply:
Finally got Style to pass, I kept missing a space in my phpdoc...